earth_enterprise/rpms/build.gradle

```diff
@@ -219,7 +212,14 @@ task stageOpenGeeInstall(type: Exec, dependsOn: 'compileOpenGee') {
         "installdir=${stagedInstallDir}")
 }
 
-task openGeePostGisRpm(type: GeeRpm) {
+if (project.hasProperty('buildOpenGee')) {
+    setGeeVersion.dependsOn stageOpenGeeInstall
+}
+
+task openGeePostGisRpm(type: GeeRpm, dependsOn: [setGeeVersion]) {
+    doFirst {
+        requires('opengee-common', ospackage.version, GREATER | EQUAL)
+    }
```

Comment:
Things that we can't set during the config phase but need to be set before the task is run we can just create an action closure to the front of the action queue for that task.  As long as these things are done before the default actions are run for that task type the default actions don't really care if those things where done in the config phase or an action at the head of the tasks' action queue.  Ether way is good.